src/common/Tile/GenreSection/index.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
+import { useSelector } from "react-redux";
 import GenreTile from "../GenreTile";
 import { StyledSection } from "./styled";
+import { selectGenres } from "../../../features/movies/moviesSlice"
 
-const GenreSection = ({ horizontal, genres }) => (
-  <StyledSection horizontal={horizontal}>
-    {genres.map((genre) => (
-      <GenreTile key={genre} horizontal={horizontal} genre={genre} />
-    ))}
-  </StyledSection>
-);
+const GenreSection = ({ horizontal, genres }) => {
+
+  const genresList = useSelector(selectGenres);
+
+  return (
+    <StyledSection horizontal={horizontal}>
+      {genresList.genres && genres?.map((genre) =>
```

Comment:
Ja mam tylko pytanie, bo zastanawiam się nad kwestią czemu tutaj masz genres?.map, a nie po prostu genres.map ???

Reply:
To jest coś czego Krzysiek w kursie nie pokazał a moim zdaniem to jest bardzo użyteczne,
dodanie "?" to jest takie zastąpienie, if,
czyli inaczej trzeba byłoby napisać, if genres true to coś, a tak to ten "?" załatwia sprawę, czyli jeśli genres są to pójdzie dalej a jeśli nie to nie wyświetli błędu